conanfile.py

```diff
@@ -0,0 +1,53 @@
+from conans import ConanFile, CMake, tools
+
+
+class HelloConan(ConanFile):
+    name = "LibConfig"
+    version = "1.7.2"
```

Comment:
Don't hardcode this.
You can use [`set_version`](https://docs.conan.io/en/latest/reference/conanfile/methods.html#set-name-set-version) to parse the version from `configure.ac` with a nice regex.